ceph_deploy/osd.py

```diff
@@ -466,11 +466,8 @@ def disk_list(args, cfg):
 
 
 def osd_list(args, cfg):
-    # FIXME: this portion should probably be abstracted. We do the same in
-    # mon.py
-    cfg = conf.ceph.load(args)
-    mon_initial_members = cfg.safe_get('global', 'mon_initial_members')
-    monitors = re.split(r'[,\s]+', mon_initial_members)
+
+    monitors = mon.get_mon_member(args)
 
     if not monitors:
         raise exc.NeedHostError(
```

Comment:
If you pass the flag to raise an exception on error to get_mon_members, the additional check and raise can be removed.